website/docs/r/firewall_rule.markdown

```diff
@@ -36,7 +36,7 @@ The following arguments are supported:
 
 * `zone` - (Optional) The DNS zone to which the Firewall Rule should be added. Will be resolved to `zone_id` upon creation.
 * `zone_id` - (Optional) The DNS zone to which the Filter should be added.
-* `action` - (Required) The action to apply to a matched request. Allowed values: "block", "challenge", "allow", "js_challenge".
+* `action` - (Required) The action to apply to a matched request. Allowed values: "block", "challenge", "allow", "js_challenge", "log".
```

Comment:
Let’s call out that log is Enterprise specific to avoid any confusion with other plan types trying to use this value.

Reply:
I've changed the wording to be `Enterprise plan also allows "log".`